### PR TITLE
CVSL-1429 repackaging jobs and removing deprecated endpoint

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/ComController.kt [hunk 1]
```diff
@@ -69,47 +67,6 @@ class ComController(private val comService: ComService, private val notifyServic
     this.comService.updateComDetails(body)
   }
 
-  @Tag(name = Tags.JOBS)
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/ComController.kt:72]: extracted into own class

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt [hunk 2]
```diff
@@ -763,85 +763,6 @@ class LicenceController(
...
-  @Tag(name = Tags.JOBS)
-  @Deprecated(
-    message = "This endpoint is being deprecated as it is believed to no longer be in use after the activation job was moved to the API",
-  )
-  @PostMapping(value = ["/activate-licences"])
-  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
-  @Operation(
-    summary = "Activate licences in bulk",
-    description = "Set licence statuses to ACTIVE. Accepts a list of licence IDs. Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
-    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt:775]: Deleted as no longer used

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt [hunk 3]
```diff
@@ -763,85 +763,6 @@ class LicenceController(
...
-  ) {
-    licenceService.activateLicencesByIds(request)
-  }
-
-  @Tag(name = Tags.JOBS)
-  @PostMapping(value = ["/inactivate-licences"])
-  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
-  @Operation(
-    summary = "Inactivate licences in bulk",
-    description = "Set licence statuses to INACTIVE. Accepts a list of licence IDs. Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt:812]: extracted into own class

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt [hunk 4]
```diff
@@ -512,13 +512,6 @@ class LicenceService(
     }
   }
 
-  @Transactional
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt:515]: no longer used

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt (hunk 2 above)
Comment (gurps-singh) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt:775]: I take it there aren't any controller tests associated with this route which also need to be deleted?

**Review by gurps-singh — APPROVED** [review]

**Pull request merged**